src/error.rs

```diff
@@ -32,9 +32,9 @@ pub enum Error {
     /// The `dlclose` call failed and system did not report an error.
     DlCloseUnknown,
     /// The `LoadLibraryW` call failed.
-    LoadLibraryW { source: WindowsError },
+    LoadLibraryExW { source: WindowsError },
```

Comment:
Consider not removing the old variants quite yet, as it makes what could be a non-breaking change a breaking one.